From aa511e0a10e08a316716f616f8073efb99c35fb9 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Sat, 6 Jul 2019 01:07:04 +0000
Subject: [PATCH] joyent/sdc-imgapi-cli#1 no workie with npm 3: `Error: Cannot
 find module 'sdc-clients/node_modules/restify'` joyent/sdc-imgapi-cli#2
 Setting JOYENT_IMGADM_IDENTITY doesn't seem to work...

---
 README.md | 1 +
 1 file changed, 1 insertion(+)

diff --git a/README.md b/README.md
index 1a9ed34..d0544ce 100644
--- a/README.md
+++ b/README.md
@@ -1,5 +1,6 @@
 # sdc-imgapi-cli
 
+
 This repository is part of the Joyent Triton project. See the [contribution
 guidelines](https://github.com/joyent/triton/blob/master/CONTRIBUTING.md) --
 *Triton does not use GitHub PRs* -- and general documentation at the main
-- 
2.21.0

